ddl/job_table.go

```diff
@@ -173,6 +183,41 @@ func (d *ddl) getJob(se *sess.Session, tp jobType, filter func(*model.Job) (bool
 	return nil, nil
 }
 
+func hasSysDB(job *model.Job) bool {
+	sNames := job2SchemaNames(job)
```

Comment:
Just a suggestion: we can avoid the slice allocation by passing a checker function(`func(tableName string) bool`) into `job2SchemaNames`.

Reply:
I will use check ID instead of name in the next PR. Also will update it.